frontend/src/concepts/pipelines/content/__tests__/ComputeRunStatusUtils.spec.tsx

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable camelcase*/
+import {
+  BanIcon,
+  CheckCircleIcon,
+  ExclamationCircleIcon,
+  NotStartedIcon,
+  QuestionCircleIcon,
+  SyncAltIcon,
+} from '@patternfly/react-icons';
+import React from 'react';
+import { PipelineRunStatusesKF, RunStorageStateKF } from '~/concepts/pipelines/kfTypes';
+import { computeRunStatus } from '~/concepts/pipelines/content/utils';
+
+const run = {
```

Comment:
Try to always assign a type:
```suggestion
const run: PipelineRunKF = {
```